wasmbinding/custom_querier.go

```diff
@@ -94,6 +95,40 @@ func CustomQuerier(qp *QueryPlugin) func(ctx sdk.Context, request json.RawMessag
 			}
 
 			return bz, nil
+
+		case contractQuery.FullDenom != nil:
+			creator := contractQuery.FullDenom.CreatorAddr
+			subdenom := contractQuery.FullDenom.Subdenom
+
+			fullDenom, err := GetFullDenom(creator, subdenom)
+			if err != nil {
+				return nil, sdkerrors.Wrap(err, "unable to get full denom")
+			}
+
+			res := bindings.FullDenomResponse{
+				Denom: fullDenom,
+			}
+
+			bz, err := json.Marshal(res)
+			if err != nil {
+				return nil, sdkerrors.Wrap(err, "failed to JSON marshal FullDenomResponse response.")
+			}
+
+			return bz, nil
+
+		case contractQuery.DenomAdmin != nil:
+			res, err := qp.GetDenomAdmin(ctx, contractQuery.DenomAdmin.Subdenom)
+			if err != nil {
+				return nil, err
+			}
+
+			bz, err := json.Marshal(res)
+			if err != nil {
+				return nil, fmt.Errorf("failed to JSON marshal DenomAdminResponse response: %w", err)
```

Comment:
You use `sdkerrors.Wrap` everywhere above, but here you use `fmt.Errorf`. Why?

Reply:
It was copypasted from https://github.com/osmosis-labs/osmosis/blob/dcc5b61fcbf5e3715c01da9268e82b1791e82257/wasmbinding/query_plugin.go#L84